<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GolfClass</class>
 <widget class="QMainWindow" name="GolfClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Golf</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>801</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="scorecard_tab">
     <attribute name="title">
      <string>Scorecard</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Select golfer</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_sel_golfer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="currentText">
       <string/>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::InsertAlphabetically</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Select course</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_sel_course">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::InsertAlphabetically</enum>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>313</x>
        <y>-51</y>
        <width>20</width>
        <height>621</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse scorecard</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_reg_scorecard">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date of round</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tbl_show_strokes">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>30</y>
        <width>91</width>
        <height>491</height>
       </rect>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="rowCount">
       <number>18</number>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>50</number>
      </attribute>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>Strokes</string>
       </property>
      </column>
     </widget>
     <widget class="QListWidget" name="list_show_scorecards">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>301</width>
        <height>481</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableWidget" name="tbl_sel_strokes">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>91</width>
        <height>491</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>18</number>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>50</number>
      </attribute>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>Strokes</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Register scorecard</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="date_sel_time">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="register_tab">
     <attribute name="title">
      <string>Register</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>New golfer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Handicap</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spin_reg_handicap">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>54</number>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>-30</y>
        <width>31</width>
        <height>611</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>0</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>New course</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_reg_course">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>510</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_reg_golfer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tbl_reg_holes">
      <property name="geometry">
       <rect>
        <x>205</x>
        <y>101</y>
        <width>561</width>
        <height>391</height>
       </rect>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="rowCount">
       <number>18</number>
      </property>
      <property name="columnCount">
       <number>6</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>90</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>Par</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Handicap</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Length</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Water</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Fairway bunker</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Green bunker</string>
       </property>
      </column>
     </widget>
     <widget class="QTextEdit" name="txt_reg_golferName">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>101</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="txt_reg_courseName">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>50</y>
        <width>231</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Golf.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_date"/>
  <buttongroup name="buttonGroup_course"/>
  <buttongroup name="buttonGroup_name"/>
 </buttongroups>
</ui>
